package.json

```
@@ -16,17 +16,20 @@
     "start": "webpack serve --mode development --open"
   },
   "devDependencies": {
-    "eslint": "8.13.0",
-    "eslint-config-htmlacademy": "5.1.2",
     "@babel/core": "7.17.9",
     "@babel/preset-env": "7.16.11",
     "babel-loader": "8.2.4",
     "copy-webpack-plugin": "10.2.4",
+    "eslint": "8.13.0",
+    "eslint-config-htmlacademy": "5.1.2",
     "webpack": "5.72.0",
     "webpack-cli": "4.9.2",
     "webpack-dev-server": "^4.8.1"
```

Все зависимости должны быть зафиксированы
```suggestion
    "webpack-dev-server": "4.8.1"
```